snull/snull.c

```diff
@@ -520,7 +520,7 @@ int snull_tx(struct sk_buff *skb, struct net_device *dev)
 		len = ETH_ZLEN;
 		data = shortpkt;
 	}
-	dev->trans_start = jiffies; /* save the timestamp */
+	//dev->trans_start = jiffies; /* save the timestamp */
 
```

Comment:
if the line is not needed, then delete it.

Reply:
Hi, I have deleted the line.